bzt/resources/selenium_extras.py

```diff
@@ -145,3 +152,74 @@ def get_locator(self, locators):
         # restore the implicit wait value
         self.driver.implicitly_wait(self.timeout)
         return locator
+
+
+class WaitForManager:
+
+    def __init__(self, driver, timeout=30):
+        self.driver = driver
+        self.timeout = timeout
+        self.loc_mngr = LocatorsManager(driver, timeout)
```

Comment:
is it really necessary to have two LocatorsManagers in test class? (one as member and one more as 'member of member (WaitForManager))

Reply:
I can think of a different solution... Or do you have a better idea?